static/src/stylesheets/module/email-signup/_form.scss

```diff
@@ -254,6 +254,18 @@
     }
 }
 
+.email-sub__form--thrasher-documentaries {
+    .email-sub__thrasher-embed-label {
+        color: $brightness-7;
+    }
```

Comment:
That would work too and would save a few lines - I don't mind either way but I think its slightly less error-prone to have all of the modifiers styles for `.email-sub__form--thrasher-documentaries` in one block rather than having the style for the documentaries label in one block and the button in another.